partials/forms/textarea-group.html

```diff
@@ -0,0 +1,17 @@
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} validation-error{{/error}}">
+    <label for="{{id}}" class="form-label-bold{{#hiddenLabel}} visuallyhidden{{/hiddenLabel}}">
+        {{#error}}<span class="error-message" aria-hidden="true">{{error.message}}</span>{{/error}}
+        {{{label}}}
+        {{#hint}}<span {{$hintId}}id="{{hintId}}" {{/hintId}}class="form-hint">{{hint}}</span>{{/hint}}
+        {{#error}}<span class="visuallyhidden">{{error.message}}</span>{{/error}}
+    </label>
+    <textarea
+        name="{{id}}"
+        id="{{id}}"
+        class="form-control{{#className}} {{className}}{{/className}}{{#error}} invalid-input{{/error}}"
+        aria-required="{{required}}"
+        {{#maxlength}} maxlength="{{maxlength}}"{{/maxlength}}
```

Comment:
except that it is: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea